strr-host-pm-web/app/stores/hostOwner.ts

```diff
@@ -89,11 +89,27 @@ export const useHostOwnerStore = defineStore('host/owner', () => {
     owner.value.lastName = userInfo.lastName
   }
 
-  const validateOwner = (): boolean => {
-    if (!hasHost.value || !hasCompParty.value) {
-      return false
+  const validateOwners = (returnBool = false): MultiFormValidationResult | boolean => {
+    let success = false
+    if (hasHost.value && hasCompParty.value) {
+      success = true
     }
-    return true
+
+    if (returnBool) {
+      return success
+    }
+    return [{
+      formId: 'host-owners',
+      success,
+      errors: [
+        ...(!hasCompParty.value
+          ? [{ message: 'Missing completing party', code: 'custom', path: ['owners.completingParty'] } as z.ZodIssue]
```

Comment:
Do we need i18n here?

Reply:
I don't think its necessary because they aren't shown in the UI (would only ever see them in the logs)